Chapters/FirstContact/FirstContactShrunk.md

```diff
@@ -32,15 +32,13 @@ In this case, we have three buttons and a text screen.
 
 The methods of the class provide the initialization and configuration of the presenters, e.g.,
 labels and actions, as well as the logic of their interaction. The
-basic design of our GUI, i.e., how the widgets are laid out, is defined by a method
-at class side.
+basic design of our GUI, i.e., how the widgets are laid out, is defined by the class as well.
```

Comment:
Defining it at the class side was the way to do it in Spec 1.0, not Spec 2.0.

Reply:
Yes